governance/votes/Executive vote - April 16, 2021.md

```diff
@@ -0,0 +1,87 @@
+* "MCD_CLIP_CALC_LINK_A": "0xbd586d6352Fcf0C45f77FC9348F4Ee7539F6e2bD" - Contract preforming the LINK-A price calculator function.
+
+Additionally, the MCD_FLIP_LINK_A contract will be removed as it was responsible for the current liquidation system. 
+
+Further details and rationale can be found in this [thread](https://forum.makerdao.com/t/mip45-liquidations-2-0-liq-2-0-liquidation-system-redesign/6352) or by browsing the [Liq2.0 Tag](https://forum.makerdao.com/tag/liq-20) in the forum.
+
+### Add LINK-A to Liquidations 2.0 Framework
+
+Given the successful outcome of this [poll](https://vote.makerdao.com/polling/QmPtNp9a?network=mainnet#poll-detail), LINK-A will be added to the Liquidations 2.0 Framework with the following parameters:
+
+**Auction Price**
+* [Auction Price Function (`calc`)](https://community-development.makerdao.com/en/learn/governance/param-auction-price-function): Stairstep Exponential
+   * [Price Change Multiplier (`cut`)](https://community-development.makerdao.com/en/learn/governance/param-auction-price-function): 0.99
+   * [Price Change Interval (`step`)](https://community-development.makerdao.com/en/learn/governance/param-auction-price-function): 90 seconds
+* [Auction Price Multiplier (`buf`)](https://community-development.makerdao.com/en/learn/governance/param-auction-price-multiplier): 1.3
+
+**Limits**
+* [Local Liquidation Limit (`ilk.hole`)](https://community-development.makerdao.com/en/learn/governance/param-local-liquidation-limit): 6 million DAI
+* [Maximum Auction Drawdown (`cusp`)](https://community-development.makerdao.com/en/learn/governance/param-max-auction-drawdown): 0.4
+* [Maximum Auction Duration (`tail`)](https://community-development.makerdao.com/en/learn/governance/param-max-auction-duration): 8,400 seconds
+* [Breaker Price Tolerance (`tolerance`)](https://community-development.makerdao.com/en/learn/governance/param-breaker-price-tolerance): 0.5
+
+**Incentives**
+* [Proportional Kick Incentive (`chip`)](https://community-development.makerdao.com/en/learn/governance/param-proportional-kick-incentive): 0.1%
+* [Flat Kick Incentive (`tip`)](https://community-development.makerdao.com/en/learn/governance/param-flat-kick-incentive): 0 DAI
+
+**Global Parameters**  
+Because this is the first vault type using liquidations 2.0, the Global Liquidation Limit will also be set:
+* [Global Liquidation Limit (`Hole`)](https://community-development.makerdao.com/en/learn/governance/param-global-liquidation-limit): 100 million DAI
+
+Further details and rationale can be found in the following [Risk Evaluation](https://forum.makerdao.com/t/link-a-liquidations-2-0-parameters/7180).
+
+### Increase Emergency Shutdown Threshold
+
+Given this [Risk Evaluation](https://forum.makerdao.com/t/informal-poll-should-we-raise-the-minimum-mkr-needed-to-call-end-cage-for-emergency-shutdown/7277/16) the following change will be made to the amount of MKR tokens needed to activate an Emergency Shutdown (call `end.cage`):
```

Comment:
I will write a forum post on this, not sure if Smart Contracts are going to do one too.

Reply:
Asked the chat again, will add if they do